CodeGen/Generators/NanoFrameworkGenerator.cs

```diff
@@ -126,6 +120,22 @@ public static void Generate(string rootDir, Quantity[] quantities)
                 GenerateQuantity(sb, quantity, Path.Combine(outputQuantitites, $"{quantity.Name}.g.cs"));
                 GenerateProject(sb, quantity, projectPath);
 
+
+                // Convert decimal based units to floats; decimals are not supported by nanoFramework
+                if (quantity.BaseType == "decimal")
+                {
+                    var replacements = new Dictionary<string, string>
+                    {
+                        //{ "(\\)sdecimal(\\s)", "$1float$2" }
+                        { "(\\d)m", "$1f" },
+                        { "(\\d)M", "$1f" },
+                        { " decimal ", " float " },
+                        { "(decimal ", "(float " }
```

Comment:
Why is float better than double? Decimal was originally chosen for a handful few quantities, to support higher precision of conversions between very large and very small units.

Reply:
Hardcoded version is fine for now, we can work more on this later.

Reply:
Honestly I don't remember why I went with float. I agree double would be better.  Changed to use double.